src/puma-cli.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+'use strict';
+const program = require('commander');
+const puma = require('pumascript');
+const fs = require("fs");
+const injector = require('./injector');
+
+let run = (req, op) => {
+    try {
+        puma.evalPuma(fs.readFileSync(req, 'Utf8'));
+    } catch (error) {
+        console.error(`ERROR [puma-cli <run>]: ${error.message}`);
+    }
+}
+
+let compile = (req, op) => {
+    let aux = req.split('.');   
+    if (!aux.slice(1, aux.length).includes('puma')) {
```

Comment:
Hi, `aux.slice(1, aux.length)` it's just going to return the file name without the first letter,
also `includes('puma')` will return false positives, like in `notapuma.file`.
Maybe you should do `aux.split('.')` and then check if that array `includes('puma')`
```js
!'file.puma.js'.split('.').includes('puma')
```

Reply:
Hi, slice returns me, as I am using it, all the elements of the arrangement minus the first one. In other words, what I do in that part of the code:
Let's suppose that:
req = "file.puma.ts"
 ``` javascript 
aux = req.split(".") // ["file", "puma", "ts"]
aux.slice(1, aux.length); // ["puma", "ts"]
```
Then I compare with includes() if it is in the array.
I based on this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

I do it this way by itself it presents the situation that the name of the file is puma.
Example: puma.exe
The name of the file is puma but the extension does not.